benchmarks/transactions-generator/src/lib.rs

```diff
@@ -130,11 +130,11 @@ impl TxGenerator {
         block_hash: &CryptoHash,
         client_sender: &ClientSender,
     ) -> bool {
-        const AMOUNT: near_primitives::types::Balance = 1_000;
+        const AMOUNT: near_primitives::types::Balance = 1; // each transaction will transfer this amount
```

Comment:
```suggestion
        // each transaction will transfer this amount
        const AMOUNT: near_primitives::types::Balance = 1; 
```

This style is considered more "rusty".